lib/unleash/context.rb

```diff
@@ -19,8 +19,11 @@ def initialize(params = {})
     end
 
     def to_s
-      "<Context: user_id=#{@user_id},session_id=#{@session_id},remote_address=#{@remote_address},properties=#{@properties}" \
-        ",app_name=#{@app_name},environment=#{@environment}>"
+      formatted_attributes = instance_variables.map do |var|
+        "#{var.to_s.delete('@')}=#{instance_variable_get(var)}"
+      end
+
+      "<Context: #{formatted_attributes.join(', ')}>"
```

Comment:
Here is the simplest possible example:
```suggestion
      "<Context: user_id=#{@user_id},session_id=#{@session_id},remote_address=#{@remote_address},properties=#{@properties}" \
        ",app_name=#{@app_name},environment=#{@environment},current_time=#{@current_time}>"
```

Reply:
Thank you very much! Simple and easy to understand. I will incorporate this suggestion!

Reply:
Commit: https://github.com/Unleash/unleash-client-ruby/pull/175/commits/0734a991dbe1d1ba68a43aeb982d2170962a7aa3